mlflow/tracking/client.py

```diff
@@ -3352,6 +3360,10 @@ def print_model_version_info(mv):
         """
         _validate_model_version_or_stage_exists(version, stage)
         if stage:
+            _logger.warning(
+                "The `stage` parameter of the `set_model_version_tag` API is deprecated. "
+                + _STAGES_DEPRECATION_WARNING
+            )
```

Comment:
`logger.warning` annoy users when it's called many times. We should use the `warnings` module instead.

Reply:
/ai give me example code

Reply:
/ai what would happen if I put the example code in a function, and use that function in other functions (e.g. f1 and f2)?

Reply:
/ai If I call `f1` and `f2`, how many times do I see the warning?

Reply:
Cool! I'll use `warnings` instead.